api/common.go

```diff
@@ -16,22 +16,28 @@ import (
 
 // Common constants for daemon and client.
 const (
-	// Current REST API version
+	// Version of Current REST API
 	Version version.Version = "1.20"
 
-	// Minimun REST API version supported
+	// MinVersion represents Minimun REST API version supported
 	MinVersion version.Version = "1.12"
 
-	// Default filename with Docker commands, read by docker build
+	// DefaultDockerfileName is the Default filename with Docker commands, read by docker build
 	DefaultDockerfileName string = "Dockerfile"
 )
 
-type ByPrivatePort []types.Port
+// byPrivatePort is a temporary type used for sort with sort.Sort() func.
```

Comment:
I'm trying to explain why byPrivatePort would have methods `Len()`，`Swap()` and `Less()`
It's OK for me to remove the O(nlog(n)) stuff since it's not that necessary. 
Updated.